account-kit/react/src/components/auth/card/index.tsx

```diff
@@ -114,6 +114,7 @@ export const AuthCardContent = ({
     } else if (authStep.type !== "initial") {
       didGoBack.current = false;
     } else if (!didGoBack.current && isAuthenticating) {
+      openAuthModal();
```

Comment:
should this step always be `email_completing`? how is the oauth step with redirect handled?

Reply:
I was confused about that too- I think at the moment, oauth with redirect also puts it in an "email_completing" state. That's a bug because it will have the wrong text, but it doesn't affect the demo app. I'll make a ticket for it.